rpc/rpc_client.go

```diff
@@ -253,7 +253,7 @@ func Connect(connConfig Config, suppressRegister bool, dispatcherFuncs map[strin
 
 	clientSingleton.Conns = values.Config().RPCPoolSize
 	if clientSingleton.Conns == 0 {
-		clientSingleton.Conns = 20
```

Comment:
Should this be less than 1? Rather than 0?

Reply:
@zalbiraw when it would be less than 0?